portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,16 +32,23 @@
 /** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  int maxNumComments = 1;
   private final DatastoreService dataStore = DatastoreServiceFactory.getDatastoreService();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Comment").addSort("timestamp", SortDirection.ASCENDING);
     PreparedQuery results = datastore.prepare(query);
     List<String> myComments = new ArrayList<>();
+    int index = 0;
     for (Entity entity : results.asIterable()) {
-        myComments.add((String) entity.getProperty("text"));
+        if (maxNumComments == index) {
+            break;
+        }
```

Comment:
If you want a limit you can also use FetchOptions: `results.asList(FetchOptions.Builder.withLimit(maxNumComments))`.

Reply:
noted! used!